eng/common/scripts/Delete-RemoteBranches.ps1

```diff
@@ -30,7 +30,7 @@ foreach ($branch in $branches)
     exit 1
   }
 
-  if ($pullRequests -and (($pullRequests | ? { $_.State -eq "open" }).Count -eq 0))
+  if (($pullRequests | ? { $_.State -eq "open" }).Count -eq 0)
```

Comment:
It took me a while to parse this expression. Whilst it uses the expressive capabilities of PowerShell, I wonder if it might not be better to do something like this:
```powershell
$openPullRequests = $pullRequests | ? { $_.State -eq "open" }

if ($openPullRequests.Count -eq 0)
{
...
```

I try to keep the impressive PowerShell one-liners for when I'm trying to get a part in the next Sandra Bullock tech thriller :)